crates/macros/src/implement.rs

```diff
@@ -221,7 +221,7 @@ pub fn gen(
                         if let Some(base) = &self.base {
                             ::windows::Interface::query(base, iid, interface)
                         } else {
-                            ::windows::HRESULT(0x8000_4002) // E_NOINTERFACE
+                            ::windows::HRESULT::E_NOINTERFACE
```

Comment:
This is very tempting - indeed I did this at first - but having these constants bleed through just encourages developers to take advantage of the limited set and then complain that E_MYFAVE is missing.

Reply:
I don't think we have access to `::windows::bindings` here? Any other solution that doesn't involve hardcoding the constant?

Reply:
No, I'm not aware of another solution.